backend/tests/__init__.py

```diff
@@ -0,0 +1,41 @@
+from courses.views import CourseList, CourseListSearch
+
+
+def set_mock_get_serializer_context():
+    """
+    Sets `CourseListSearch.get_serializer_context` to `CourseList.get_serializer_context`
+    and returns the original value of `CourseListSearch.get_serializer_context`
+    This function is called in `tests.__init__.py` to ensure it takes effect for
+    all tests by default. The goal is to reduce testing runtime, though this also means
+    that the `recommendation_score` field of the `CourseListSearch` endpoint always returns
+    null (`None`) by default in all the tests. However if you would
+    like revert to the original (production) version of `CourseListSearch.get_serializer_context`
+    it is returned from this function and stored as a package variable as
+    production_CourseListSearch_get_serializer_context. To revert to the original version
+    for a single test or test case, decorate your test case or test like this:
+        ```
+        @unittest.mock.patch('courses.views.CourseListSearch.get_serializer_context',
+               new=production_CourseListSearch_get_serializer_context)
+        ```
+    (Note that you should put
+    `from tests import production_CourseListSearch_get_serializer_context`
+    at the top of the file to make production_CourseListSearch_get_serializer_context
+    is defined. Note also that reverting may lead to a `botocore`
+    error in the CI tests. To resolve this `botocore` error it is suggested that you patch
+    `plan.management.commands.recommendcourses.retrieve_course_clusters`. See
```

Comment:
This should be `courses.views.retrieve_course_clusters` (as discussed on Slack), right?

Reply:
Yes, good catch